readme.md

```diff
@@ -38,40 +38,66 @@ The Autocomplete functionality of Notepad++ is not accessible by default. The au
 Unfortunately, the keyboard shortcuts dialog is not friendly to NVDA by default. This add-on makes this dialog accessible. You can tab between the components and press the arrow keys to manipulate the controls like you would for any other dialog.
 
-### Incremental search
+### Incremental Search
 
-One of the most interesting features of notepad++ is the ability to use incremental search. 
+One of the most interesting features of Notepad++ is the ability to use incremental search. 
 Incremental search is a search mode in which you search for a phrase of test by typing in the edit field, and the document scrolls to show you the search in real time. 
 As you type, the document scrolls to show the line of text with the most likely phrase you are looking for. It also highlights the text that matched.
 The program also shows you how many matches have been detected. There are buttons to move to the next and previous match.
-As you type, NVDA will announce the line of text that notepad++ detected a search result in. NVDA also announces how many matches there are, but only if the number of matches has changed. 
+As you type, NVDA will announce the line of text that Notepad++ detected a search result in. NVDA also announces how many matches there are, but only if the number of matches has changed. 
 When you found the line of text you want, simply press escape, and that line of text will be at your cursor.
 To launch this dialog, select incremental search from the search menu, or press alt+control+i.
 
-### Reporting information about the  current line
+### Reporting Information about the Current Line
 
 Pressing nvda+shift+\ (back slash) at any time will report the following:
 
 * the line number
 * the column number I.E. how far into the line you are.
-* the selection size, (number of characters horizontally selected, followed by  the number of characters vertically selected, which would make a rectangle. This info is only reported if relevant.
+* the selection size, (number of characters horizontally selected, followed by the number of characters vertically selected, which would make a rectangle. This info is only reported if relevant.
 
-### Support for the previous/next find feature.
+### Support for the Previous/Next Find Feature
 
 By Default, if you press control+f you bring up the find dialog. 
 If you type text here and press enter, the text in the window is selected and the document is moved to the next search result. 
 In Notepad++ you can press f3 or shift+f3 to repeat the search in the forward or backward direction respectively. 
 NVDA will read both the current line, and the selection within the line which represents the found text.
 
-## Non-default Notepad++ keyboard shortcuts
+### Preview of MarkDown or Hypertext as Webpage
+
+Notepad++ does natively not support MarkDown (*.md) with e.g. language highlighting. 
```

Comment:
Nit: natively not, -> not natively

Reply:
Depends on what the emphasis lies on but your version is certainly more natural.